books/model-mongodb.js

```diff
@@ -39,12 +40,17 @@ module.exports = function(config) {
 
 
   function getCollection(cb) {
+    if (collection) {
+      setImmediate(function() { cb(null, collection); });
```

Comment:
Gotcha. That's fine, I was just a little surprised to see it. I'm still somewhat green on node as well.